packages/action-menu/stories/action-menu.stories.ts

```diff
@@ -112,6 +114,19 @@ export default {
                 type: 'boolean',
             },
         },
+        staticValue: {
+            name: 'static',
+            type: { name: 'string', required: false },
+            description: 'The visual static variant to apply to the button.',
+            table: {
+                type: { summary: 'string' },
+                defaultValue: { summary: 'none' },
+            },
+            control: {
+                type: 'select',
+                options: ['white', 'black', 'none'],
```

Comment:
I think we cover `undefined` with `"none"` if we use:
```
labels: {
          // 'labels' maps option values to string labels
          white: 'white',
          black: 'black',
          none: undefined,
        },
```
But, I've just read it straight out of the [docs](https://storybook.js.org/docs/react/essentials/controls#dealing-with-complex-values) and not used it before, so double check.

Reply:
ok